dateparser/languages/validation.py

```diff
@@ -10,7 +10,7 @@ class LanguageValidator(object):
                   'monday', 'tuesday', 'wednesday', 'thursday', 'friday', 'saturday', 'sunday',
                   'january', 'february', 'march', 'april', 'may', 'june', 'july',
                   'august', 'september', 'october', 'november', 'december',
-                  'year', 'month', 'week', 'day', 'hour', 'minute', 'second']
+                  'year', 'month', 'week', 'day', 'hour', 'minute', 'second', 'date_order']
```

Comment:
We also need to add validation method for it that would check existence of `date_order` key and correctness of its value.